dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -249,15 +249,14 @@ public void Dispose()
     /// <summary>
     /// JSON serialized string representation of the function.
     /// </summary>
-    /// <returns></returns>
     public override string ToString()
         => this.ToString(false);
 
     /// <summary>
     /// JSON serialized string representation of the function.
     /// </summary>
     public string ToString(bool writeIndented)
-       => JsonSerializer.Serialize(this, options: new JsonSerializerOptions() { WriteIndented = writeIndented });
+        => JsonSerializer.Serialize(this, options: new JsonSerializerOptions() { WriteIndented = writeIndented });
```

Comment:
This new JsonSerializerOptions must have been added recently, since I last looked for occurrences of this.  We should cache these rather than creating a new one on each call, e.g.
```C#
private static readonly JsonSerializerOptions s_indentedOptions = new() { WriteIndented = true };
...
public string ToString(bool writeIndented) => writeIndented ?
    JsonSerializer.Serialize(this, s_indentedOptions) :
    JsonSerializer.Serialize(this);
```